src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -178,9 +180,10 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
             
             var hostName = string.IsNullOrEmpty(dynamoModel.HostName) ? "Dynamo" : dynamoModel.HostName;
 
-            string buildId = "", releaseId = "";
-            Version version;
-            if (Version.TryParse(dynamoModel.Version, out version))
+            hostInfo = new HostContextInfo() { ParentId = dynamoModel.UpdateManager.ParentId, SessionId = dynamoModel.UpdateManager.SessionId };
```

Comment:
We only reference `HostContextInfo` here so Analytics.Net refrence is kept only for DynamoCore